provisioning/transport/mqtt/src/ProvisioningChannelHandlerAdapter.cs

```diff
@@ -271,6 +279,30 @@ private Task PublishRegisterAsync(IChannelHandlerContext context)
             return context.WriteAndFlushAsync(message);
         }
 
+        private async Task VerifyPublishPacketTopicAsync(IChannelHandlerContext context, string topicName, string jsonData)
+        {
+            Match match = RegistrationStatusTopicRegex.Match(topicName);
+            if(match.Groups.Count >= 2)
+            {
+                if(Enum.TryParse(match.Groups[1].Value, out HttpStatusCode statusCode))
+                {
+                    if (statusCode >= HttpStatusCode.BadRequest)
+                    {
+                        var errorDetails = JsonConvert.DeserializeObject<ProvisioningErrorDetails>(jsonData);
+
+                        bool isTransient = statusCode >= HttpStatusCode.InternalServerError || (int)statusCode == 429;
```

Comment:
It can't parse 429 because it's not part of the enum? What is the string that the server would return in that case?